lib/organisms/gravity-form/parsers/form.data.js

```diff
@@ -0,0 +1,93 @@
+angular
```

Comment:
The gulp task that generates the requires for all js files does not support subfolders inside the main folder of each component nowadays. We should add this feature to the gulp task (I'll open an issue).